appsdk/sdk.go

```diff
@@ -327,6 +327,8 @@ func (sdk *AppFunctionsSDK) Initialize() error {
 		}
 	}
 
+	sdk.appCtx, sdk.appCancelCtx = context.WithCancel(context.Background())
```

Comment:
Why moved? Don't see it used prior to previous location. Add comment why it needs to be here.

Reply:
ok, will add comments- the reason is to have skd.appCtx populated and available for the call happening in line 388 `sdk.initalizeSecretProvider()`, in which NewSecret will take context as input argument.